test/test_vcard_wrapper.py

```diff
@@ -463,7 +463,7 @@ def test_setting_and_getting_webpage_ablabel(self):
         wrapper._add_webpage({'github': 'https://github.com/scheibler/khard'})
         wrapper._add_webpage('http://example.com')
         self.assertListEqual(wrapper.webpages, [
-            'github: https://github.com/scheibler/khard',
+            {'github': 'https://github.com/scheibler/khard'},
             'http://example.com'])
 
     @unittest.expectedFailure
```

Comment:
If you claim to fix #221 you have to fix this test somehow.

Reply:
(and check for other candidates)

Reply:
If there's a reason this needs to not be a dict (or somewhere I haven't expanded it) let me know and I'll have another look. This seemed to be the neatest way of keeping all the data between functions.

Reply:
I see what you meant now. github's interface didn't help by hiding the test you were talking about!

Reply:
This checks for a list because `VcardWrapper.organisation` is documented to be a list. I just improved the docstring of `_add_organisation`, check out 3447d658b784e29ca4e5f45dd88ee6c7335cca7e.

Reply:
Ah *looks*

Reply:
Is there any reason I can't change the relevant docstrings? to `list(list(str)) or list(dict(list(str)))`?

Reply:
I might need to return to this tomorrow to get the types all straight in my head.

Reply:
Also, I suppose I should probably use `add_labelled_object` for any field using `_get_multi_property`, and adjust their docstrings accordingly

Reply:
Ok. `TITLE`, `ROLE`, `NICKNAME`, and `NOTE` now use `_add_labelled_object`, and I've changed their docstrings and some others to reflect the appropriate `:rtype`s.

If I'm going in the wrong direction, let me know.